src/many-abci/src/many_app.rs

```diff
@@ -24,10 +24,16 @@ pub struct AbciModuleMany<C: Client> {
     backend_status: base::Status,
     identity: CoseKeyIdentity,
     backend_endpoints: BTreeMap<String, EndpointInfo>,
+    allow_addrs: BTreeSet<Address>,
```

Comment:
Isn't it more accurate to have `Option<BTreeSet<Address>>`? An empty allow_addrs set could mean we don't allow anyone to write, for example, which is different than allowing everyone to write.